quantus/helpers/model/pytorch_model.py

```diff
@@ -97,6 +98,19 @@ def _get_model_with_linear_top(self) -> torch.nn:
 
         return linear_model
 
+    def _obtain_predictions(self, x, model_predict_kwargs):
+        pred = None
+        if isinstance(self.model, PreTrainedModel):
+            if not isinstance(x, BatchEncoding):
+                raise ValueError(
+                    "When using HuggingFace pretrained models, please use Tokenizers output for `x`"
+                )
+            pred = self.model(**x, **model_predict_kwargs).logits
```

Comment:
Should we also enable softmax here (post accessing the logits)? so that we convert the pred to softmax, if softmax=True? (we can add it as a class attribute above).

Reply:
I implemented this way, please see if you agree: [9a67c4c](https://github.com/understandable-machine-intelligence-lab/Quantus/pull/339/commits/9a67c4c933784c3db962e9969bf74f4b409a12de)